docs/hpc/03_storage/01_intro.md

```diff
@@ -1 +1,15 @@
-# Available storage systems
+# HPC Storage
+
+The NYU HPC clusters are served by a General Parallel File System (GPFS) cluster. 
+
+The NYU HPC team supports data storage, transfer, and archival needs on the HPC clusters, as well as collaborative research services like the Research Project Space (RPS).
+
+Team is also piloting all flash VAST storage.
```

Comment:
We run VAST in production now :)

Reply:
Ah, thanks for catching that @s-sajid-ali !
How about I remove line 7 and replace line 3 with:
The NYU HPC clusters are served by a General Parallel File System (GPFS) cluster and an all Flash VAST storage cluster.

How does that sound?

Reply:
Sounds good.

Reply:
Great, thanks!  I'll make the change.